ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,70 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
+        name (string): Object name.
```

Comment:
can be removed; only resource-specific args should be set along with `kwargs`

Reply:
Ack